### PR TITLE
Flesh out documentation for next release

### PR DESCRIPTION
**Pull request by LPGhatguy**
I struggled writing this documentation a bit. The interaction between different configuration bits are very tangled and the formatting I chose ended up being awkward.

We should also expand the example projects to include features that don't have examples yet, like `includes` and both styles of code generation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,64 +13,169 @@
...
+
+```bash
+cargo install --git https://github.com/rojo-rbx/tarmac
+```
+
+## Basic Example
+**The [examples](examples) folder contains small, working projects using different features from Tarmac.**
 
-To get started, make a `tarmac.toml` file in the root of your project. This is where you can configure where Tarmac will look for assets and what it will do with them.
+Tarmac is configured by a [TOML](https://github.com/toml-lang/toml) file in the root of a project named `tarmac.toml`. This is where you can configure where Tarmac will look for assets and what it will do with them.
```
Comment (ZoteTheMighty) [inline README.md]: Second sentence feels a bit awkward here.
```suggestion
Tarmac is configured by a [TOML](https://github.com/toml-lang/toml) file in the root of a project named `tarmac.toml`. Tarmac uses this file to determine where to look for assets and what to do with them.
```

File: README.md [hunk 2]
```diff
@@ -13,64 +13,169 @@
...
+	* A list of additional paths to search for additional projects in. The inputs from discovered projects will be merged into this project, and other settings ignored.
+
+### InputConfig
+* `glob`, string
+	* A path glob that should include any files for this input group.
+	* Tarmac uses the [globset library](https://docs.rs/globset/0.4.5/globset/) and supports any syntax it supports.
+* `codegen`, bool, **optional**
+	* Whether Tarmac should generate Lua code for the assets contained in this input group. Defaults to **false**.
+* `codegen-path`, path, **optional**
+	* If defined and `codegen` is true, Tarmac will merge all generated Lua code for this input group into a single file.
```
Comment (ZoteTheMighty) [inline README.md:172]: I think you might be right about this being kind of unintuitive. I wonder if we can restructure these fields in such a way that the one-script-per-asset behavior seems less like the default?

**Review by ZoteTheMighty — COMMENTED** [review]
Looks good overall

File: README.md (hunk 2 above)
Comment (LPGhatguy) [reply README.md:172]: I think I'll follow up with another change later to simplify config overall.

**Pull request merged**